dxvk.conf

```diff
@@ -581,3 +570,12 @@
 # - True/False
 
 # dxvk.enableDebugUtils = False
+
+# Memory limit for locked D3D9 resources
```

Comment:
Probably mention that it's only relevant for 32bit. So people won't get confused.

Reply:
I don't really want users to touch that anyway.